File: Acceso a Datos/WorkSpace Eclipse/5EjercicioObjetos/src/Persona.java
import java.io.Serializable;

public class Persona implements Serializable {

	private static final long serialVersionUID = -5046577359557334793L;
	String nombre;
	int edad;
	int telefono;

	public Persona(String nombre, int edad, int telefono) {
		this.nombre = nombre;
		this.edad = edad;
		this.telefono = telefono;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public int getEdad() {
		return edad;
	}

	public void setEdad(int edad) {
		this.edad = edad;
	}

	public int getTelefono() {
		return telefono;
	}

	public void setTelefono(int telefono) {
		this.telefono = telefono;
	}

	@Override
	public String toString() {
		return "Nombre: " + nombre + "\nEdad: " + edad + "\nTelefono: " + telefono;
	}

}

File: Sistemas de Gestión Empresarial/WorkSpace PyCharm/EjerciciosClase/09 Clases y Métodos/Persona.py
class Persona:
    nombre = "";
    edad = 0;
    ciudad = "";

    def __init__(self, nombre, edad, ciudad):
        self.nombre = nombre;
        self.edad = edad;
        self.ciudad = ciudad;

    def get_nombre(self):
        return self.nombre;

    def get_edad(self):
        return self.edad;

    def get_ciudad(self):
        return self.ciudad;

    def set_nombre(self, nombre):
        self.nombre = nombre;

    def set_edad(self, edad):
        self.edad = edad;

    def set_ciudad(self, ciudad):
        self.ciudad = ciudad;

    def toString(self):
        return self.get_nombre() + " " + str(self.get_edad()) + " " + self.get_ciudad() + ".";


persona = Persona("Iván", 19, "León");
print(persona.toString());
